pkgs/by-name/ad/adbtuifm/package.nix

```
@@ -0,0 +1,29 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  android-tools,
+  lib,
+}:
+buildGoModule rec {
+  pname = "adbtuifm";
+  version = "v0.5.8";
```

```suggestion
  version = "0.5.8";
```

The version must start with a digit

https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming